internal/test_helpers/course_definition.yml

```
@@ -748,3 +748,7 @@ stages:
 
     marketing_md: |
       In this stage, you'll add support for encoding the response body using `gzip`.
+
+  - slug: "pe1"
+  - slug: "pe2"
+  - slug: "pe3"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)